### PR TITLE
Add some notes to the readme for developer setup

### PR DESCRIPTION
**Pull request by carver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,32 @@ The shadowlands installer is hosted under the ['shadowlands'](https://github.com
...
+     hidapi/libusb/hid.c:47:10: fatal error: libusb.h: No such file or directory
+     #include <libusb.h>
+              ^~~~~~~~~~
+    compilation terminated.
+    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
+
+Install the header files with:
+
+```sh
+sudo apt-get install libusb-1.0-0-dev libudev-dev
```
Comment (kayagoban) [inline README.md]: git, build-essential, autoconf, libtool, pkg-config, libusb-dev, libudev-dev, libssl-dev, libusb-1.0-0-dev, bash, xsel, xclip, and python > 3.5.3 are the dependencies I use in the debian package.

Comment (carver) [reply README.md]: Are `xsel`, `xclip`, `bash` actually required, or just preferences?

**Comment by kayagoban** [comment]
Bash wouldn’t be needed for dev contributors using only shadowlands-core - that’s currently only needed for the loading script/installer.

xsel/xclip are needed for copying to the clipboard, which is core functionality.

**Comment by carver** [comment]
I assume libtool and pkg-config are only required for building the package, so left them out. Happy to add it back in though, if you think they are important for running locally.

**Comment by kayagoban** [comment]
That sounds right.  If it’s wrong, somebody can put in an issue.

**Pull request merged**